beam-postgres/Database/Beam/Postgres/PgSpecific.hs

```diff
@@ -1409,14 +1410,14 @@ pgUnnestArray (QExpr q) =
   fmap (\(PgUnnestArrayTbl x) -> x) $
   pgUnnest' (\t -> emit "UNNEST" <> pgParens (fromPgExpression (q t)))
 
-data PgUnnestArrayWithOrdinalityTbl a f = PgUnnestArrayWithOrdinalityTbl (C f Int) (C f a)
+data PgUnnestArrayWithOrdinalityTbl a f = PgUnnestArrayWithOrdinalityTbl (C f Int64) (C f a)
   deriving Generic
 instance Beamable (PgUnnestArrayWithOrdinalityTbl a)
 
 -- | Introduce each element of the array as a row, along with the
 -- element's index
 pgUnnestArrayWithOrdinality :: QExpr Postgres s (V.Vector a)
-                            -> Q Postgres db s (QExpr Postgres s Int, QExpr Postgres s a)
+                            -> Q Postgres db s (QExpr Postgres s Int64, QExpr Postgres s a)
```

Comment:
How are deciding which Int type to use?

Reply:
Postgres documentation: https://www.postgresql.org/docs/12/queries-table-expressions.html

Reply:
Oh I see it's the `bigint` here in the docs for `WITH ORDINALITY`... I wish there were a more obvious way to connect this choice with the docs.

Reply:
I'm normally not a big fan of type aliases but perhaps it makes sense to define `type PgBigInt = Int64` to make it more obvious. But even then we have to go digging through docs again to see where the bigint shows up.

Reply:
Hmm, I think I'd rather not use type aliases. Besides all the usual reasons, it seems a bit more confusing to have something like `type PgInt = Int32`.

Reply:
Yeah I'm not really a fan either. It's just that there's not a very mechanical way to verify these kinds of choices strewn throughout the codebase. Perhaps that's just par for the course.